### ISSUE TITLE
[benchmarks][wasm] Test correctness of MobileNetV3 is false

### ISSUE THREAD

**Issue by qjia7** [tensorflow/tfjs#6867]
Steps to reproduce:
1. Open https://tensorflow.github.io/tfjs/e2e/benchmarks/local-benchmark/index.html 
2. Choose `MobileNetV3` and `wasm` backend, click `Test correctness`.

Expect `Prediction matches CPU` is `true`. Actually got `false`.
OS: windows
browser: 105.0.5195.127

**Comment by mattsoulanille** [comment]
I can reproduce this on Chrome 106.0.5249.91 linux.

**Comment by mattsoulanille** [comment]
This is due to XNNPack not supporting a kernel size of 1x1 in AvgPool (which is pretty reasonable since that turns it into the identity function, not counting padding and strides). I'm working on a fix, but I need to account for strides and padding.

Why does MobileNetV3 have an AvgPool node with a 1x1 kernel that behaves as the identity function? Maybe there's something wrong in our conversion? Specifically, the node I'm talking about is `StatefulPartitionedCall/StatefulPartitionedCall/predict/MobilenetV3/Logits/AvgPool`.

**Comment by mattsoulanille** [comment]
Fixed by #6969. Confirmed with the local benchmark tool.
